### ISSUE TITLE
Won't start after installing another package

### ISSUE THREAD

**Issue by albertdyu** [matthewcarlucci/DiscoRhythm#7]
Hello Dr. Carlucci,

I was a happy DiscoRhythm user right up until today, when I installed the GenomicFeatures package and now discoApp() returns the following error message:

The `name` provided ('cloud-upload') is deprecated in Font Awesome 5:
* please consider using 'cloud-upload-alt' or 'fas fa-cloud-upload-alt' instead
* use the `verify_fa = FALSE` to deactivate these messages
The `name` provided ('arrows') is deprecated in Font Awesome 5:
* please consider using 'arrows-alt' or 'fas fa-arrows-alt' instead
* use the `verify_fa = FALSE` to deactivate these messages
This Font Awesome icon ('clock-o') does not exist:
* if providing a custom `html_dependency` these `name` checks can 
  be deactivated with `verify_fa = FALSE`
Error: shinyjs: extendShinyjs: `functions` argument must not contain any of the following function names:
reset

Removing GenomicFeatures doesn't fix the problem, so I'm assuming I updated something bad along the way. Would you happen to have any ideas what's going wrong?

Best regards,

Albert

**Comment by matthewcarlucci** [comment]
Hi Albert, thanks for reporting this. It seems to be due to breaking changes to shiny 1.7 (https://www.biostars.org/p/9490625/). A temporary fix is to downgrade shiny by:

```r
library(devtools)
install_version("shiny", version = "1.6.0", repos = "http://cran.us.r-project.org")
```

I will investigate a proper fix by using the fontawesome package.

Matt

**Comment by albertdyu** [comment]
Thank you! I had to downgrade Shinyjs as well, but that did indeed do the trick!

**Comment by matthewcarlucci** [comment]
DiscoRhythm 1.10.1 is now available which fixed this with ddf2cbf51684fb6f5b07d6baaeb07ecf2fb7903a
